src/instructlab/model/evaluate.py

```diff
@@ -23,7 +23,7 @@
 
 if typing.TYPE_CHECKING:
     # Third Party
-    from instructlab.eval.evaluator import Evaluator
+    pass
```

Comment:
Should we just remove this check altogether if we're just passing it?

Reply:
Thanks, looks like isort made the change.  Fixing.

Reply:
Fixed!